python/bufr/obs_builder/get_default_nprocs.py

```diff
@@ -0,0 +1,39 @@
+
+import os
+
+def get_default_nprocs(default=8):
+    """
```

Comment:
Would it be better to call this function something like `nprocs_per_task`?

Reply:
Oh, yours is better, so I changed.  Done!